src/main/webapp/completedPath.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name = "viewpoint" content="width=device-width, initial-scale=1">
+
+  <title>COMPSCI:GO</title>
+
+  <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+  <script src="completedPath.js"></script>
+</head>
+<body class="step-img-4" onload="loadFunctions()">
+  <h2 class="step-outline">THE END?</h2>
+  <img class="step-pixel-professor" src="images/graduationTeacher.png" alt="pixel teacher">
+
+  <audio id="musicPlayer" src="music/graduationMusic.mp3" autoplay loop></audio>
+  <a href="#" class="step-align-iconv2" id="musicIcon" onclick="playmusic()"></a>
+
+  <div class="step-wrap, step-dialogue">
+      <div>
```

Comment:
Does the end game page need dialogue and experience? This page is linked to after the user finishes clicking through all the game stages in the path.

Reply:
It does need dialogue but i don't think it needs experience unless we do like a littel showcase of this is all you hav achieved. I think this can b eimplemented in a diffrent PR

Reply:
I would do this. Put a TODO with an issue about it